benchmarks/perf-tool/okpt/io/config/parsers/test.py

```diff
@@ -23,6 +23,7 @@ class TestConfig:
     test_name: str
     test_id: str
     endpoint: str
+    port: int
```

Comment:
If port is configurable we need to update description of test params in benchmarks/perf-tool/README

Reply:
ack.